public/components.html

```diff
@@ -0,0 +1,588 @@
+                        </button>
+                        <button class="inline-button ">
+                            <img height="30px " src="../images/remove-from-cart.png " alt=" ">
+                                Remove from Cart
+                        </button>
+                        <button class="inline-button ">
+                            <img height="30px " src="../images/purchase.png " alt=" ">
+                                Purchase
+                        </button>
+                        <button class="inline-button ">
+                            <img height="40px " src="../images/add-to-wishlist.png " alt=" ">
+                                Add to wishlist
+                        </button>
+                    </div>
+                </div>
+            </div>
+
+            <hr>
+
+            <!-- Gird Layout -->
+            <div id="grids">
+                <h1 class="card-heading">Grid Layout <button class="myBtn_multi code-btn"><i class="fas fa-code "></i></button> </h1>
+                <div class="teal-card">
+                    <div class="grid-container">
+                        <div class="item1">Header</div>
+                        <div class="item2">Menu</div>
+                        <div class="item3">Main</div>
+                        <div class="item4">Right</div>
+                        <div class="item5">Footer</div>
+                    </div>
+                </div>
+            </div>
+
+            <hr>
+
+            <!-- Cards -->
+            <div id="cards">
+                <h1 class="card-heading ">Cards</h1>
+                <div class="offwhite-card ">
+                    <h2 class="sub-heading ">Checkout Card <button class="myBtn_multi code-btn"><i class="fas fa-code "></i></button></h2>
+                    <div class="product-card ">
+                        <div class="product-img "></div>
+                        <div class="product-desc ">
+                            <div class="product-information ">
+                                <h2 style="font-weight: lighter ">LI-NING XP-90-IV Black, White, Silver</h2>
+                                <span>Delivery in 3-4 days | </span>
+                                <span style="color:green;font-weight:bold ">Free</span>
+                                <s>₹40</s>
+                                <p>Pack of:1, 86 g</p>
+                                <div style="display: flex; ">
+                                    <p style="margin-right:10px ">Seller: RetailNet </p>
+                                    <img height="20px " src=" ../images/fassured.png " alt=" ">
+                                </div>
+                                <h3>₹350</h3> <span> <s>₹990</s></span>
+                                <strong style="color:green ">64% Off 1 offer available </strong>
+                                <br>
+                                <h2 style="display:block; font: size 1rem;margin-top:1.4rem; ">REMOVE</h2>
```

Comment:
decrease the font size and make it button instead of h2 heading

Reply:
Done